skins/tailwindui/layouts/auth-split.php

```diff
@@ -22,7 +22,9 @@
             </div>
 
             <div class="hidden lg:block relative w-0 flex-1">
-                <img class="absolute inset-0 h-full w-full object-cover" src="<?= e(Url::asset(config('brand.backgroundImage'))); ?>" alt="">
+            <?php if ($background): ?>
+                <img class="absolute inset-0 h-full w-full object-cover" src="<?= e($background); ?>" alt="">
```

Comment:
Where is the $background value coming from?

Reply:
it's been fixed.